gitinfo

```diff
@@ -0,0 +1 @@
+{"commit":"a7f39c8e0ca94f4e73055d40653e0af0a7bf3773","time":"2024-06-25T13:50:15+02:00","branch":"DALA-4303_Company_Contact_Details"}
```

Comment:
Ich glaube diese Datei wurde mal versehentlich gepusht.
Kannst du sie bitte löschen?

Reply:
removed

Reply:
Das "gitinfo" file ist immer noch da, wenn ich mir den branch anschaue:

https://github.com/d-fine/Dataland/tree/DALA-4303_Company_Contact_Details

Reply:
it doesn't show gitinfo for me when I see the diff of this branch and main. I think it was pushed in an earlier commit, then I reverted it in the "adjustments based on review comments" commit. The content should be identical to main

Reply:
Ja aber ich glaube wir wollen das File gar nicht in main haben. Das muss versehentlich passiert sein in irgendeinem PR.

Daher kannst du das File ruhig komplett rauslöschen. Dann wirst du auch einen diff zu main sehen, und zwar, dass das File in main existiert, bei dir im feature branch aber nicht.

Reply:
removed